build/Targets/VSL.Versions.targets

```diff
@@ -17,6 +17,18 @@
     <BuildNumberBuildOfTheDayPadded Condition="('$(BuildNumber)' != '') AND ($(BuildNumber.Split('.').Length) == 2)">$(BuildNumber.Split('.')[1].PadLeft(2,'0'))</BuildNumberBuildOfTheDayPadded>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
We don't maintain any `.props` file in this repo. If we are to introduce one then we will need to move lot more than just this one. Our need here is simple, unlike Roslyn, so I think we can stick with this.